source/Ship.cpp

```diff
@@ -2660,6 +2670,9 @@ double Ship::JumpDriveFuel() const
 	if(!attributes.Get("jump drive"))
 		return 0.;
 	
+	if(HasOutfit("jump drive", "scram drive"))
+		return BestFuel("jump drive", "scram drive", 175.);
```

Comment:
Why would a jump drive with more abilities than a normal one have a lower default fuel? 250 makes more sense to me than 175.